src/status_im/contexts/profile/settings/screens/password/change_password/new_password_form.cljs

```diff
@@ -115,17 +101,21 @@
     [:<>
      [rn/scroll-view {:style style/form-container}
       [header/view]
-      [password-with-hint
-       {:hint           {:text   (i18n/label :t/password-creation-hint)
-                         :status (if long-enough? :success :default)
-                         :shown? true}
+      [password-with-hint/view
+       {:hint           (if (not short-enough?)
+                          {:text   (i18n/label :t/password-creation-max-length-hint)
+                           :status :error
+                           :shown? true}
+                          {:text   (i18n/label :t/password-creation-hint)
+                           :status (if long-enough? :success :default)
+                           :shown? true})
```

Comment:
Here we're showing the "maximum 100 characters" error for the Change Password form.